plonetheme/onegovbear/theme/javascript/tinyMCEIntegration.js

```diff
@@ -0,0 +1,40 @@
+/*
+TinyMCE Dialog is loaded in an iFrame so it's not possible
+to override the stlying. So this script copies the related
+stylings to the head section of the iframe.
+TODO: Not working on page load.
```

Comment:
Any idea how we can add support for non-overlay instances of tinymce?
Is there an event fired after tinymce is initialized, maybe? E.g. in Plone's TinyMCE integration?